src/services/login/AuthService.js

```diff
@@ -35,7 +35,7 @@ const encryptSecret = testKey => {
  * @param {string} key key in jwt.keys.json
  * @returns String or null if key is not found
  */
-const decryptSecret = key => {
+export const decryptSecret = key => {
```

Comment:
```suggestion
const decryptSecret = key => {
```
Can you ctrl+shift+f and check if this is used anywhere? It shouldn't be exported because despite our shitty security, we're going to only try and use it inside AuthService lol